styles.css

```diff
@@ -1,3 +1,90 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+h1 {
```

Comment:
try to be careful about global type targetting like this. you should prefer classes and IDs over types